src/Application/Mappings/AutoMaperConfig.cs

```diff
@@ -14,7 +15,22 @@ public static void RegisterAutomapper(this IServiceCollection services)
 
     public static IMapper Initialize() => new MapperConfiguration(cfg =>
     {
-      cfg.CreateMap<Product, GetProductDto>();
-    }).CreateMapper();
+        cfg.CreateMap<Product, GetProductDto>();
+
+        cfg.CreateMap<CreateProductDto, Product>()
```

Comment:
Ta mapa jest niepotrzebna, skoro mamy korzystać z fabryki produktów, to takie mapowanie wprost nie będzie używane